apps/modernization-ui/src/generated/graphql/schema.ts

```diff
@@ -372,6 +373,88 @@ export type InvestigationFilter = {
   programAreas?: InputMaybe<Array<InputMaybe<Scalars['String']['input']>>>;
   providerFacilitySearch?: InputMaybe<ProviderFacilitySearch>;
 };
+export type CreateQuestionFormType = {
+  __typename?: 'CreateQuestionFormType';
+  defaultLabelInReport?: Scalars['String']['input'];
+  dataMartColumnName?: Scalars['String']['input'];
+  defaultRdbTableName?: Scalars['String']['input'];
+  datamartColName?: Scalars['String']['input']; //?
+  rdbColumnName?: Scalars['String']['input'];
+  reportLabel?: Scalars['String']['input'];
+  codeSystem?: Scalars['String']['input'];  //code
+  conceptCode?: Scalars['String']['input'];  //code
+  conceptName?: Scalars['String']['input'];  //code
+  fieldLength?: Scalars['Int']['input'];  //code
+  mask?: CreateNumericQuestionRequest.mask | any;  //code
+  minValue?: Scalars['Int']['input'];  //code
+  maxValue?: Scalars['Int']['input'];  //code
+  allowFutureDates?: Scalars['Boolean']['input'];
+  dateFormat?: InputMaybe<Scalars['Int']['input']>;
+  preferredConceptName?: Scalars['String']['input'];  //code
+  adminComments?: Scalars['String']['input'];  //code
+  codeSet?: CreateNumericQuestionRequest.codeSet;  //code
+  description?: Scalars['String']['input'];  //code
+  defaultValue?: Scalars['String']['input'];  //code
+  label?: Scalars['String']['input'];  //code
+  displayControl?: Scalars['Int']['input'];  //code
+  displayType?: Scalars['String']['input'];  //code
+  subgroup?: Scalars['String']['input'];  //code
+  tooltip?: Scalars['String']['input'];  //code
+  uniqueName?: Scalars['String']['input'];  //code
+  uniqueId?: Scalars['String']['input'];  //code
+  includedInMessage?:Scalars['Boolean']['input'];  //?
+  requiredInMessage?:Scalars['Boolean']['input'];  //?
+  messageVariableId?: Scalars['String']['input'];
+  messageLabel?: Scalars['String']['input'];
+  hl7DataType?: Scalars['String']['input'];
+  HL7Segment?: Scalars['String']['input'];
+  relatedUnits?: Scalars['String']['input'];
+};
```

Comment:
Issue (Blocking): This seems to be manually created code within the `generated` folder for `graphQL`. There should be no need to create a `CreateQuestionFormType` as the various types for creation of a question should be generated from the API.

See:
[CreateNumericQuestionRequest](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/page-builder/generated/models/CreateNumericQuestionRequest.ts)

[CreateDateQuestionRequest](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/page-builder/generated/models/CreateDateQuestionRequest.ts)

[CreateTextQuestionRequest](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/page-builder/generated/models/CreateTextQuestionRequest.ts)

[CreateCodedQuestionRequest](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/page-builder/generated/models/CreateCodedQuestionRequest.ts)